website/content/docs/nextjs.md

```diff
@@ -0,0 +1,228 @@
+
+export default class Home extends Component {
+  render() {
+    let { html , attributes:{ title, cats } } = content;
+    return (
+      <article>
+          <h1>{title}</h1>
+          <div dangerouslySetInnerHTML={{ __html: html }}/>
+          <ul>
+              { cats.map((cat, k) => (
+                  <li key={k}>
+                    <h2>{cat.name}</h2>
+                    <p>{cat.description}</p>
+                  </li>
+              ))}
+          </ul>
+      </article>
+    )
+  }
+}
+
+```
+Great! We now have a page that displays content from our Markdown file. Go ahead and start your development server to test if everything is working:
+
+```bash
+npm run dev
+```
+
+## Adding Netlify CMS
+
+There are many different ways to add Netlify CMS to your project. The easiest is probably just to embed it from a CDN, and that's exactly what we're gonna do. To avoid making this guide too complicated, we're just going to add Netlify into a subfolder inside the ```/static``` directory (which is just served as static files by Next):
+
+```bash
+# Create and navigate into static/admin folder
+mkdir static/admin
+cd static/admin
+
+# Create index.html and config.yml file
+touch index.html
+touch config.yml
+```
+
+Paste HTML for Netlify CMS into your ``static/admin/index.html`` file (check out the [Add Netlify To Your Site](http://localhost:8000/docs/add-to-your-site/) section for more information)
+
+```html
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Content Manager</title>
+  <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+</head>
+<body>
+  <!-- Include the script that builds the page and powers Netlify CMS -->
+  <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
+</body>
+</html>
+```
+Notice that we also added the identity widget. This allows sign up when the project is hosted at Netlify.
+
+Paste the following configuration into your```static/admin/config.yml``` file:
+
+```yaml
+backend:
+  name: git-gateway
+  branch: master
+media_folder: static/img
+collections:
+  - name: "pages"
+    label: "Pages"
+    files:
+    - label: "Home"
+      name: "home"
+      file: "content/home.md"
+      fields: 
+        - { label: "Title", name: "title", widget: "string"}
+        - { label: "Publish Date", name: "date", widget: "datetime" }
+        - { label: "Body", name: "body", widget: "markdown"}
+        - label: 'Cats'
+          name: "cats"
+          widget: list
+          fields: 
+            - { label: "Name", name: "name", widget: "string"}
+            - { label: "Description", name: "description", widget: "text"}
+```
+
+Awesome! Netlify CMS should now be available at ```localhost:3000/static/admin/index.html```.
+Unfortunatly we can't edit our content just yet. First we need to move our code into a git repository, and create a new Netlify site.
```

Comment:
Unfortunately

Reply:
Great, thanks for your feedback! I just fixed the typo 😁